.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ on: # yamllint disable-line rule:truthy rule:comments
   pull_request: ~
 
 env:
-  APP_NAME: "nautobot-app-dev-example"
+  APP_NAME: "nautobot-dev-example"
```

Comment:
We need to make sure to fix this in any apps that get the new dockerfile. This needs to match the image in the `development/docker-compose.dev.yml`